common/src/main/org/jetbrains/lincheck/descriptors/CodeLocations.kt

```diff
@@ -12,14 +12,17 @@ package org.jetbrains.lincheck.descriptors
 
 import org.jetbrains.lincheck.trace.TraceContext
 import java.util.Objects
+import kotlin.String
+import kotlin.collections.List
```

Comment:
I think this stuff is imported by default

Reply:
yup